pkg/terraform/plan.go

```diff
@@ -45,6 +46,10 @@ func planArgsFromOptions(opts *PlanOptions) []string {
 		args = append(args, "-compact-warnings")
 	}
 
+	if util.PtrVal(opts.Destroy) {
```

Comment:
As a follow-up, we should switch to https://github.com/abcxyz/pkg/blob/main/pointer/pointer.go and drop the util package. Filed https://github.com/abcxyz/guardian/issues/356.

Reply:
Ack, will change in a follow up (didnt know that existed)